### DISCUSSION TITLE
Error with refresh names in Layer Manager

### DISCUSSION THREAD

**Discussion by JGrancha** [GrapesJS/grapesjs#4047] [Q&A] [answered]
Good afternoon and merry christmas.

Before opening an issue I have been browsing a lot and trying various things that have not worked for me. I am changing the "name" property of the model with a trait. The problem comes when I open the layer viewer.
When changing the property and opening the layer viewer the name is displayed correctly.
Once I have opened the layer viewer, if I change the "name" again it no longer refreshes. If I move the component from the layer viewer then the name is updated. Is there a way to force the layer viewer to refresh?

Default model:

![image](https://user-images.githubusercontent.com/5517910/148238486-2e72fa59-b7a4-4645-bb80-cfc0431af24a.png)

Type:

![image](https://user-images.githubusercontent.com/5517910/148238430-8e5fb234-e612-449d-88aa-ad0c53e0bbdf.png)

I am attaching a video showing what happens
![ERROR_GRAPESJS](https://user-images.githubusercontent.com/5517910/148236492-fc9c5b4a-5139-44e1-8c49-573ab09f5173.gif)
.

**Comment by JGrancha** [comment]
@artf Do you have any idea if it is a grapejs error or is there something I am not doing right?

**Comment by artf** [comment, marked as answer]
Yeah, that could be fixed.
For the moment I'd suggest triggering this post name update:
```js
// ...
this.set('name', '...');
this.trigger('rerender:layer');
```

**Reply by JGrancha** [reply]
Yeah, it works!! Thanks!
